bin/build.sh

```diff
@@ -32,7 +32,7 @@ echo "==> Building indico wheel"
 # indico/bin/maintenance/build-wheel.py all-plugins "${PLUGINS_DIR}"
 
 echo "==> Building distribution wheel"
-indico/bin/maintenance/build-wheel.py plugin --no-git "${ROOT_DIR}"
+uv run indico/bin/maintenance/build-wheel.py plugin --no-git "${ROOT_DIR}"
```

Comment:
I would rather have the `build.sh` script be run with `uv run` in the same way that the maintenance scripts inside the `indico` submodule assume they will be run from inside a virtualenv. 

I can still see value in adding `uv run` in the `make build` target.